ci-operator/config/redhat-developer/rhdh/.claude/skills/rhdh-eks-lifecycle/scripts/check-eks-lifecycle.sh

```
@@ -0,0 +1,75 @@
+
+set -euo pipefail
+
+EKS_DOCS_URL="https://raw.githubusercontent.com/awsdocs/amazon-eks-user-guide/mainline/latest/ug/versioning/kubernetes-versions.adoc"
+EOL_API_URL="https://endoflife.date/api/amazon-eks.json"
+MAPT_REF=""
+TEST_PATTERN=""
+CONFIG_DIR="ci-operator/config/redhat-developer/rhdh"
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --mapt-ref)       MAPT_REF="$2"; shift 2 ;;
```

Shellcheck (0.11.0)

[warning] MAPT_REF appears unused. Verify use (or export if used externally).

(SC2034)